tools/rsdl/rapid-cli/rapid.rdm.transformation.tests/data/type-facets/sample.csdl.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+  <edmx:DataServices>
+    <Schema Namespace="Model" xmlns="http://docs.oasis-open.org/odata/ns/edm">
+      <ComplexType Name="A" OpenType="true">
+        <Property Name="prop00" Type="Edm.String" Nullable="false" />
+        <Property Name="prop01" Type="Edm.String" Nullable="false" MaxLength="9" />
+        <Property Name="prop02" Type="Edm.String" MaxLength="9" />
+        <Property Name="prop03" Type="Collection(Edm.String)" Nullable="false" MaxLength="9" />
+        <Property Name="prop04" Type="Collection(Edm.String)" MaxLength="9" />
+        <Property Name="prop05" Type="Edm.Decimal" Nullable="false" Scale="Variable" />
```

Comment:
```suggestion
        <Property Name="prop05" Type="Edm.Decimal" Nullable="false" Precision="34" Scale="floating" />
```

See https://github.com/oasis-open/odata-rapid/issues/162

Reply:
The OData library currently doesn't support creating Scale facet value of "Variable" .
Woudl it be ok for the moment to generate 
` <Property Name="prop05" Type="Edm.Decimal" Nullable="false" />` for a plain `Decimal` without the facet values ?

Reply:
No Precision and Scale="variable" would also work.

Reply:
The library currently also doesn't allow me to produce  ` Type="Edm.Decimal"  Scale="variable" />`
Sorry, I will file a feature request against the library.